src/collections/_documentation/platforms/dotnet/winforms.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Windows Forms
+sidebar_order: 30
+---
+
+Besides the [main configuration like calling `SentrySdk.Init`](/platforms/dotnet/), you must consider the WinForms specific configuration.
+
+Sentry's .NET SDK works with WinForms applications through the [Sentry NuGet package](https://www.nuget.org/packages/Sentry). It works with WinForms apps running on .NET Framework 4.6.1, .NET Core 3.0 or higher.
+
+## Configuration
+
+The SDK automatically captures unhandled exceptions that hit the `AppDomain.UnhandledException`, but on Windows Forms, for production apps (when no debugger is attached), the small Window pops up when a crash happens. In order to get the exception to rethrow and be captured by Sentry, you must also configure:
```

Comment:
suggest: "The SDK automatically handles `AppDomain.UnhandledException`. On Windows Forms, for production apps (when no debugger is attached), the small Window pops up when an unhandled error occurs. To get the exception to rethrow the error so it is captured by Sentry, also configure:"